src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/index.stories.tsx

```diff
@@ -19,20 +48,7 @@ export default {
   args: {
     address: AddressFactory().one(),
   },
-  render: (args) => {
-    fetchMock.get('http://localhost:8071/api/v1.0/credit-cards/', []);
-    fetchMock.get('http://localhost:8071/api/v1.0/orders/', []);
-    fetchMock.get('http://localhost:8071/api/v1.0/addresses/', []);
-    fetchMock.get(
-      'http://localhost:8071/api/v1.0/products/AAA/?course_code=BBB',
-      ProductFactory().one(),
-    );
-    return StorybookHelper.wrapInApp(
-      <QueryClientProvider client={createTestQueryClient({ user: UserFactory().one() })}>
-        <CourseProductItem {...args} />
-      </QueryClientProvider>,
-    );
-  },
+  render: (args) => render(args),
```

Comment:
```suggestion
  render,
```

Reply:
i did it at first but i had a type issue so a made it verbose.
```
Conversion of type '{ component: ({ productId, course, compact }: CourseProductItemProps) => React.JSX.Element | null; parameters: { docs: { source: { code: string; }; }; }; args: { address: Address; }; render: (args: CourseProductItemProps, options?: Maybe<...>) => React.JSX.Element; }' to type 'ComponentAnnotations<ReactRenderer, CourseProductItemProps>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Types of property 'render' are incompatible.
    Type '(args: CourseProductItemProps, options?: Maybe<{    order: CredentialOrder;}>) => React.JSX.Element' is not comparable to type 'ArgsStoryFn<ReactRenderer, CourseProductItemProps>'.
      Types of parameters 'options' and 'context' are incompatible.
        Property 'order' is missing in type 'StoryContext<ReactRenderer, CourseProductItemProps>' but required in type '{ order: CredentialOrder; }'.ts(2352)

```